### PR TITLE
13.3.0 release

### PR DESCRIPTION
**Pull request by iche033**
# 🎈 Release

Preparation for 13.3.0 release.

Comparison to 13.2.0: https://github.com/gazebosim/sdformat/compare/sdformat13_13.2.0...prep_13.3.0

* https://github.com/gazebosim/gz-sensors/pull/289
* https://github.com/gazebosim/gz-sim/issues/1865

## Checklist
- [ ] Asked team if this is a good time for a release
- [ ] There are no changes to be ported from the previous major version
- [ ] No PRs targeted at this major version are close to getting in
- [ ] Bumped minor for new features, patch for bug fixes
- [ ] Updated changelog
- [ ] Updated migration guide (as needed)
- [ ] Link to PR updating dependency versions in appropriate repository in [gazebo-release](https://github.com/gazebo-release) (as needed): <LINK>

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by ahcorde — APPROVED** [review]

File: Changelog.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
 ## libsdformat 13.X
 
-### libsdformat 13.3.0 (2022-12-12)
+### libsdformat 13.3.0 (2022-02-07)
```
Comment (azeey) [inline Changelog.md:3]: nit: 2023

File: Changelog.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+
+1. Use `File.exist?` for Ruby 3.2 compatibility
+    * [Pull request #1216](https://github.com/gazebosim/sdformat/pull/1216)
+
+1. Make ThrowOrPrintError a free internal function
+    * [Pull request #1221](https://github.com/gazebosim/sdformat/pull/1221)
+
+1. Make throwOrPrintErrors a member of sdf::Error
```
Comment (azeey) [inline Changelog.md:11]: nit: These two can be combined

**Review by azeey — APPROVED** [review]
Couple of nits. LGTM otherwise.

File: Changelog.md (hunk 1 above)
Comment (iche033) [reply Changelog.md]: fixed. [941b2c3](https://github.com/gazebosim/sdformat/pull/1229/commits/941b2c37866d208ba5ec2ed294fe2a3cb44ebec0)

File: Changelog.md (hunk 2 above)
Comment (iche033) [reply Changelog.md]: combined, and left the PR description to be `Make ThrowOrPrintError a free internal function`

File: Changelog.md [hunk 3]
```diff
@@ -1,6 +1,28 @@
...
+    * [Pull request #1217](https://github.com/gazebosim/sdformat/pull/1217)
+
+1. update Param calls to use error vectors parameters
+    * [Pull request #1140](https://github.com/gazebosim/sdformat/pull/1140)
+
+1. ign -> gz Migrate Ignition Headers : sdformat
+    * [Pull request #1118](https://github.com/gazebosim/sdformat/pull/1118)
+
+1. Don't assume CMAKE_INSTALL_*DIR variables are relative
+    * [Pull request #1190](https://github.com/gazebosim/sdformat/pull/1190)
```
Comment (scpeters) [inline Changelog.md:25]: we can remove this one too because it only applied to `sdf9` and couldn't be merged forward

**Review by scpeters — APPROVED** [review]

File: Changelog.md (hunk 3 above)
Comment (scpeters) [reply Changelog.md]: https://github.com/gazebosim/sdformat/pull/1229/commits/19cd95ebf75fea7db7ffe22444fffa4eb91a54fb

**Pull request merged**